testing/deprecated_test.py

```diff
@@ -69,9 +69,9 @@ def test_terminal_reporter_writer_attr(pytestconfig):
     assert terminal_reporter.writer is terminal_reporter._tw
 
 
-@pytest.mark.parametrize("plugin", ["catchlog", "capturelog"])
+@pytest.mark.parametrize("plugin", ["catchlog", "capturelog", "faulthandler"])
```

Comment:
import the list from the package

Reply:
the test below could be enhanced by parsing the config with parseconfig and then just using `pytest.warns`

Reply:
Good suggestions, done. 👍